### PR TITLE
Budget app

### PR DESCRIPTION
**Pull request by svitalis123**
## I accomplished

- Creating a Ruby on Rails application that allows the user to:

- [ ] register and log in, so that the data is private to them.
- [ ] introduce new transactions associated with a category.
- [ ] see the money spent on each category.

- [Here](https://www.loom.com/share/18e90d545a954fcd96b594c8b101f9a2) is my Video on the app
(https://www.loom.com/share/18e90d545a954fcd96b594c8b101f9a2)

# [here](https://menyera.onrender.com) is my deployment link
(https://menyera.onrender.com)

### CODE REVIEW

File: spec/views/groups/index.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
```
Comment (codecaiine) [inline spec/views/groups/index.html.erb_spec.rb:2]: - [ ] I think the splash screen is missing . According to the project requirement , you should have a Splash screen which is a simple page with your app name with links to the sign up and log in pages. Please fix it to make your app follow requirement :+1: 

- [ ] Good job so far you app is well implemented :clap: But there is some differences between your design and these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) :cry:  you can check the screenshots below :arrow_down:  Please make sure that your design looks similar to given design by using the same font face, size and weight, composition and space between for all your page elements as required for this project :+1: 

#### Your design
Screenshot_20230224_002439
Screenshot_20230224_002537
Screenshot_20230224_002642


#### Expected design

Screenshot_20230224_002951

Screenshot_20230224_003017
Screenshot_20230224_003029
Screenshot_20230224_003039

File: README.md [hunk 2]
```diff
@@ -117,9 +88,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://last-3f0w.onrender.com/users/1/groups)
```
Comment (codecaiine) [inline README.md]: - [ ] You app doesn't show up :broken_heart: Please make sure to fix your deployment and make your app works as expected :+1:

Comment (codecaiine) [inline README.md]: - [ ] I noticed that your video link is too long around 7 minutes . The requirement for the video length  is 5 minutes maximum . Please fix it make sure that the new video follows all the videos project requirement :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @svitalis123 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great project implementation :heavy_check_mark: 
- App work as expected :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (afanimartin) [reply README.md]: - The link https://last-3f0w.onrender.com throws the error `Not found`. Please fix that

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @svitalis123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 👍🏿 👍🏿 👍🏿 
- Good job on this project. You recorded a video presenting the project, readme is professional and the given design was followed. Good job!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @svitalis123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉 🎉 🎉 
- Changes made as requested 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**